src/components/useFetch.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react'
+
+const useFetch = (url: string) => {
+    type post = {
+        userId: number;
+        id: number;
+        title: string;
+        body: string;
+    }
+    const [data, setData] = useState<Array<post>>([])
```

Comment:
You need to clarify array here.
```suggestion
    const [data, setData] = useState<Array<post[]>>([])
```

Reply:
I'm sorry, but I don't understand. I thought I had already defined it correctly as an array of post-type objects. Or am I wrong?